database/src/migrations/20210107132219_setup_biohub_db.ts

```diff
@@ -0,0 +1,59 @@
+import Knex from 'knex';
+import * as fs from 'fs';
+
+const DB_SCHEMA1 = 'biohub';
+const DB_SCHEMA2 = 'biohub_dapi_v1';
+const DB_USER = 'biohub_api';
+const path = require('path');
```

Comment:
should avoid using require in typescript projects:

`import path from 'path';`